NuRadioMC/utilities/medium.py

```diff
@@ -1,112 +1,237 @@
-from __future__ import absolute_import, division, print_function
-import numpy as np
-from NuRadioReco.utilities import units
+from NuRadioMC.utilities.medium_base import*
+
+"""
+1) When implementing a new model it should at least inherit from
+'IceModel' from the module 'medium_base'. Overwrite all the function. 
+Inheritance from daughter classes like 'IceModel_Simple' is also 
+possible and overwriting functions may not be needed in this case.
+
+2) When implementing a new model and using the radiopropa numerical
+tracer, do not forget to implement scalar field of the refractive index
+also in the c++ code of radiopropa for a fast simulation. Implement the 
+model in IceModel.cpp and IceModel.h. Then edit the function to get the
+radiopropa ice model, so it can be used in NuRadioMC. For example
+
+        def get_ice_model_radiopropa(self):
+            scalar field = radiopropa.New_IceModel(*args)
```

Comment:
I did this because depending on which icemodel you use or module you add the arguments are different and I didn't wanted it to be to specific but I don't know how to make this make clear that these things are placeholders for the real arguments